assets/css/_sass/website/_home.scss

```diff
@@ -62,27 +65,32 @@
 
 .overview-content {
   p {
-    font-size: $base-font-size;
+    font-size: $base-font-size !important;
     font-weight: $light-weight;
     color: $grey900;
     text-align: center;
     line-height: $base-line-height !important;
-    max-width: none;
+    max-width: 43em;
+    margin: auto;
   }
   .main-logo {
     display: block;
     position: relative;
-    margin: 80px auto 28px auto;
+    margin: 80px auto 0px auto;
     img {
       height: 64px;
+      margin: 0px auto 0px auto;
     };
   }
+  .center {
+    margin: auto;
+  }
   .card {
     position: relative;
     border: 1px solid $grey400;
     margin: 32px auto 40px auto;
     padding: 0px 10px;
-    min-height: 292px;
+    min-height: 377px;
```

Comment:
Ty's design had the card at a min-height of 292px, or am I misreading the spec?

Reply:
I'll revert this one. Was trying to figure out a way to line the containers to the same height. Probably better if I look deeper into the card as a whole pattern to get the proper sizing. Maybe this is one you want to handle since I'm not super familiar with the way the card pattern as a whole was constructed.

Reply:
TBH, it's kind of an in-the-meantime fix until we get cards from Helix. I'm good with leaving this change in; I mostly was trying to understand why the 292px from the spec didn't exactly work with the build.

Reply:
Ok keeping this in the interim. This will be looked at by whoever takes the JIRA for aligning the cards and centering.